db/migrate/20230203074700_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, references: :users, null: false, foreign_key: { to_table: :users }
+      t.references :post, references: :posts, null: false, foreign_key: { to_table: :posts }
```

Comment:
- [ ] Again here the foreign ke should be set to true and not point to the users table, like the following:
`t.references :post, foreign_key: true`


### Note that this applies for the comments migration file as well